### PR TITLE
WOKE Stonekeep (Unisex Ruler and "Consort" Role)

### PR DESCRIPTION
**Pull request by lavillastrangiato**
## About The Pull Request

- The role of "King" is now unisex, with "Queen" being its female title. Queen receives everything the King does, like a typical ruler-queen would, it's just the same role. In order to not kill a bunch of people's characters, because wow you guys sure do love your elf queens, "King"/Ruler role can now be elves and half-elves. Use this wisely. Or don't, I'm not your community manager.
- The role of "Queen" has been changed to "Consort" and has become unisex as well. They also receive **classes** on roundstart, which give them the opportunity to pick a broad-strokes backstory and skills to go with it.
- "Queen"/Consort can now no longer ready up as Ruler.
- Slightly rewrites the Ruler/Consort/Hand descriptions.
- Changes the names of some keys and some files for clarity.

THE CLASSES:

HIGHBORN: Basically your average noble. Good at swords and reading. Receives the Golden-blooded trait.
COURTESAN: Ex-prostitute turned ruler's favoured consort. Good at wrestling, knives, and sneaking. Receives the Fabled Lover trait. Inspired by Inki.
LOWBORN: A peasant plucked from serfdom to be a royal spouse, for some reason or another. Good physical stats and a good crafter, but receives INT and SPD malus. Has the Seed Knower trait (this may change).

## Why It's Good For The Game

This is going to be a little long-winded, so bear with me.

In the aftermath of making the game able to start without a ruler and the Hand being able to ready as a "regent", @BadAtThisGame302 suggested that I make Queen a latejoin role. It got me thinking; that would pretty easy to implement, but I'd have to fiddle a bit with the roles. Later, @Cadet00 suggested that the role of Ruler be unisex, and The Role Formerly Known as Queen become something like a "consort", which inspired this PR.

The reason I followed through on it is that we basically have a unisex ruler role anyway, and the only thing separating them is a few skills (which aren't really that consequential), a title (easily editable depending on gender), and some clothing choices. I also heard from a few players that they would have liked the opportunity to play a male consort to a ruling Queen, and to me this seemed like the most logical and fat-trimming way to implement it.

Right now, the way this is coded in its draft state means that you can technically be a ruling couple in a same-sex relationship. I don't really care one way or another, some people like their IC homophobia, some people just wanna have gay sex, I personally think that a consort being the same sex as their ruler is more interesting than gender-locking the roles based on who plays what.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/modules/jobs/job_types/roguetown/nobility/consort.dm [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+	shoes = /obj/item/clothing/shoes/roguetown/boots
+	backr = /obj/item/storage/backpack/rogue/satchel
+	id = /obj/item/clothing/ring/silver
+	beltl = /obj/item/keyring/consort
+	beltr = /obj/item/storage/belt/rogue/pouch/coins/rich
+
+/* ! ! ! CONSORT CLASSES ! ! !
+- Highborn: The "default" class, a typical Enigman noble. Decent with swords and a knife. Can sew and read pretty good. A little squishy.
+- Courtesan: Ex-classy or not-so-classy prostitute. Concerningly good with that knife and sneaking around. A little dumb.
+- Peasant: A good wholesome peasant spouse. Can cook and swing a pitchfork good. Not so smart or perceptive.
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/roguetown/nobility/consort.dm:49]: OoOoOooooOoooh you will replace this subclass that doesn't make sense with one from Grenzelhoft oooOoOooOoooOooh

File: code/modules/jobs/job_types/roguetown/nobility/consort.dm [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+	category_tags = list(CTAG_CONSORT)
+
+/datum/outfit/job/roguetown/consort/zybantine/pre_equip(mob/living/carbon/human/H)
+	. = ..()
+	if(H.gender == MALE)
+		neck = /obj/item/clothing/neck/roguetown/horus
+		pants = /obj/item/clothing/under/roguetown/tights/red
+		shirt = /obj/item/clothing/suit/roguetown/shirt/undershirt/black
+		armor = /obj/item/clothing/suit/roguetown/armor/gambeson/arming
+	else
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/roguetown/nobility/consort.dm:182]: You really cooked with this. 5 swords if old on the consort is peak... might need a tweak.

File: code/modules/jobs/job_types/roguetown/nobility/consort.dm [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+	category_tags = list(CTAG_CONSORT)
+
+/datum/outfit/job/roguetown/consort/highborn/pre_equip(mob/living/carbon/human/H)
+	. = ..()
+	if(H.gender == MALE)
+		pants = /obj/item/clothing/under/roguetown/tights/black
+		shirt = /obj/item/clothing/suit/roguetown/shirt/undershirt/black
+		armor = /obj/item/clothing/suit/roguetown/armor/gambeson/arming
+	else
+		pants = /obj/item/clothing/under/roguetown/tights/stockings/silk/random
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/roguetown/nobility/consort.dm:67]: 4 swords on the consort nooooo 5 if old honey what are you doing!

**Review by BadAtThisGame302 — CHANGES_REQUESTED** [review]

File: code/modules/jobs/job_types/roguetown/nobility/consort.dm (hunk 3 above)
Comment (lavillastrangiato) [reply code/modules/jobs/job_types/roguetown/nobility/consort.dm:67]: THAT WAS THERE BEFORE I EDITED IT

**Comment by BadAtThisGame302** [comment]
> Honestly Lord / Lady it while at it. 
> 
> King with 2 dudes in his court in a shitty little village is and always was silly

Might need a noble poll again, or even one in announcements for the whole server for that. I made one a while ago but it's outdated and it was inconclusive.

File: code/modules/jobs/job_types/roguetown/nobility/consort.dm [hunk 4]
```diff
@@ -0,0 +1,209 @@
+/datum/job/roguetown/consort
+	title = "Consort"
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/roguetown/nobility/consort.dm:2]: Lord-Consort/Lady-Consort? Or were we keeping Consort for both genders.

**Comment by lavillastrangiato** [comment]
@NPC1314 We did poll on that, and people pretty definitively wanted to remain a petty-king/queen.

**Comment by BadAtThisGame302** [comment]
> @NPC1314 We did poll on that, and people pretty definitively wanted to remain a petty-king/queen.

image

Top one is in #general bottom one is in the noble court.

**Comment by BadAtThisGame302** [comment]
But it is 3 months ago so, and things have changed. That's why I suggested a re-do.

**Comment by lavillastrangiato** [comment]
Jesus Christ I was literally about to correct myself because I remembered it differently.

But I'm not changing it to Lord/Lady because that's dumb.

File: code/modules/jobs/job_types/roguetown/nobility/consort.dm (hunk 4 above)
Comment (lavillastrangiato) [reply code/modules/jobs/job_types/roguetown/nobility/consort.dm:2]: Consort is fine. Just call them Lord/Lady.

**Comment by NPC1314** [comment]
I don't feel constrained by a split vote of a handful of guys on discord tbh, go with what inspires you and Ill approve it.
Petty King doesn't even cover it, dude got like 1 servant and 1 retainer, literal hobo, but its your PR, do what you feel is fitting, I do the same nowadays.

**Comment by lavillastrangiato** [comment]
So, for now, I think what I'm going to do is I am going to make this ready for review as it is. The core of the PR is adding a more varied amount of classes while also leaving things fairly open for interpretation and roleplay.

The Zybantine origin doesn't really add much to the class selection beyond the language gain, so I figured I might as well bite the bullet and remove it. I think if I added something like a language selection to the character setup, then it would do what I wanted with regards to adding background flavour.

**Comment by Cadet00** [comment]
> Another option is to turn the Consort class into a more general Noble or Courtier or Councillor or something and let the Ruler marry if they like. Worth thinking on.

I wonder if we could add a way for rulers to choose if they've got a consort or not in their char set up, also I'd ask we maybe do add some race restrictions to consort, mainly Aasimar and Tieflings (drow too maybe... and that one idea of making drow consort a triumph buy would be cool).

**Comment by lavillastrangiato** [comment]
Consort's already restricted to human, elf, and half-elf...

**Comment by Cadet00** [comment]
> Consort's already restricted to human, elf, and half-elf...

I'm blind, I couldn't find it but if it already is then 👍

**Comment by NPC1314** [comment]
Being forced to have a same-sex lover against your will feels like a bit of a meme, and a little icky since the ruler don't get a say. Marriages is for show or political all the time and them being loveless has always been commonplace, but a lover/consort isn't like that.

Also with ability to latejoin you get the old classic single ruler marries, then suddenly their consort arrives and starts drama. That kinda works I guess but it could be really annoying with the wrong player combo, the setup is very questionable.

Like I imagine it will go like this:
Player A (ruler) want to do some sort of courting/romance arc with player Bs character (whoever )  but then player C latejoins as consort, expecting to do the old Queen routine or whatever but tied to the Ruler, but turns out theyre a third wheel, gets salty and starts drama to focus back on themselves, ruining things for A and B and anyone participating in the arc. That sort of thing.

Unisex ruler seems good, but the consort concept tied to it not so much, its unclear how to make consort work and not be a meme.

**Comment by Cadet00** [comment]
> Being forced to have a same-sex lover against your will feels like a bit of a meme, and a little icky since the ruler don't get a say. Marriages is for show or political all the time and them being loveless has always been commonplace, but a lover/consort isn't like that.
> 
> Also with ability to latejoin you get the old classic single ruler marries, then suddenly their consort arrives and starts drama. That kinda works I guess but it could be really annoying with the wrong player combo, the setup is very questionable.
> 
> Like I imagine it will go like this: Player A (ruler) want to do some sort of courting/romance arc with player Bs character (whoever ) but then player C latejoins as consort, expecting to do the old Queen routine or whatever but tied to the Ruler, but turns out theyre a third wheel, gets salty and starts drama to focus back on themselves, ruining things for A and B and anyone participating in the arc. That sort of thing.
> 
> Unisex ruler seems good, but the consort concept tied to it not so much, its unclear how to make consort work and not be a meme.

Yeah, some players were bringing up not letting consort late join so that it's locked in at round start, it might work esp since then a ruler knows if they're married or not,

**Comment by lavillastrangiato** [comment]
Sure, I can maybe make it that consort is ready-only, but why is it a "meme" when you're "forced" into a same-sex relationship? Arguably you're "forced" into any other relationship in the current state of things.

The alternative is to code whatever mechanic Shiptest is on when you have to accept an application for someone to join a crew role, but [makes dismissive wanking gesture]

**Comment by NPC1314** [comment]
I guess its mostly to do with pivoting the relationship from a possibly political marriage to a more informal/ volontary one, and that has some possibly negative outcomes in various constellations, but it might be worth it for the upsides.

With latejoin disabled for them its possible to hash out some stuff in lobby so that helps a lot.
There is one glaring issue though, we clearly need to modify things so we can have dwarf rulers, but that has to be its own thing since it touches on the heirs etc.

**Review by NPC1314 — APPROVED** [review]

**Pull request merged**